src/components/Capture/Document.tsx

```diff
@@ -124,13 +169,35 @@ class Document extends Component<Props> {
       hasCamera,
       isPoA,
       poaDocumentType,
+      requestedVariant,
       side,
+      trackScreen,
       translate,
       uploadFallback,
       useLiveDocumentCapture,
       useWebcam,
     } = this.props
 
+    const renderFallback = isDesktop
+      ? this.renderCrossDeviceFallback
+      : this.renderUploadFallback
+
+    if (requestedVariant === 'video') {
```

Comment:
Why not
```suggestion
    if (hasCamera && requestedVariant === 'video') {
```

Reply:
I guess I missed this. Thanks for the great spot :pray: